Header header
int32 idx
int32 class_id
string label
geometry_msgs/PoseWithCovariance pose